exporter/exporterhelper/retry_sender.go

```diff
@@ -39,6 +39,25 @@ type RetrySettings struct {
 	MaxElapsedTime time.Duration `mapstructure:"max_elapsed_time"`
 }
 
+func (cfg *RetrySettings) Validate() error {
```

Comment:
Can we add some tests in `retry_sender_test.go` for these different scenarios?

Reply:
Done.